src/lava/proc/lif/models.py

```diff
@@ -209,12 +211,14 @@ class PyLifModelFloat(AbstractPyLifModelFloat):
     algorithmic prototyping, without engaging with the nuances of a fixed
     point implementation.
     """
-    s_out: PyOutPort = LavaPyType(PyOutPort.VEC_DENSE, bool, precision=1)
+    s_out: PyOutPort = LavaPyType(PyOutPort.VEC_DENSE, float)
     vth: float = LavaPyType(float, float)
 
     def spiking_activation(self):
         """Spiking activation function for LIF.
         """
+        if self.graded_spike.item():
+            return self.v * (self.v > self.effective_vth)
```

Comment:
There are multiple ways to generate a graded spike in a LIF model. This one attaches the current voltage value. Do know if this would be the most commonly used one? I have seen people use floor(v/vth), (v-vth), etc.

Reply:
This is what Paxon uses in his RF model. As a result this is the graded spike behavior in slayer. In principle it doesn't matter as they are convertible with affine transform. So the system can be made equivalent by changing the weight and bias.